tools/excel/ccf/adobe-ccf-v5_mapping_exporter.py

```
@@ -0,0 +1,47 @@
+"""
+Simple script to export a mapping from the Adobe CCF v5 PDF
+
+Simply change the "target_column_name" variable to the column name you want in the original Excel file (e.g. SOC 2 Ref#).
+You can also modify the "destination_file" variable to make the file name more consistent.
+"""
+
+import pandas as pd
```

Pylint (3.3.7)

[error] Unable to import 'pandas'

(E0401)